cmd/globals.go

```diff
@@ -109,6 +109,9 @@ var (
 	// Time when object layer was initialized on start up.
 	globalBootTime time.Time
 
+	globalActiveCred       credential
+	globalPublicCerts      []*x509.Certificate
+	globalObjCacheDisabled bool
```

Comment:
What I mean is, we should use `xlObjects.objCacheEnabled` and `globalObjCacheDisabled` should be removed